assets/src/edit-story/elements/text/util.js

```diff
@@ -44,3 +44,43 @@ export const getHandleKeyCommand = ( setEditorState ) => ( command, currentEdito
 	}
 	return 'not-handled';
 };
+
+/**
+ * Adds a <link> element to the <head> for a given font in case there is none yet.
```

Comment:
Should this be moved the font provider?

Reply:
Seems fitting 👍

Reply:
Done https://github.com/ampproject/amp-wp/pull/3998/commits/3eba1c0d98481e5ade47de8c69534961f918986b